dev-libs/qr-code-generator/qr-code-generator-1.8.0.ebuild

```
@@ -0,0 +1,33 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="QR Code Generator Library in Multiple Languages"
+
+HOMEPAGE="
+	https://github.com/nayuki/QR-Code-generator
+	https://github.com/EasyCoding/qrcodegen-cmake
+"
+
+SRC_URI="
+	https://github.com/nayuki/QR-Code-generator/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/EasyCoding/qrcodegen-cmake/archive/v${PV}-cmake2.tar.gz -> qr-code-generator-cmake-${PV}.tar.gz
+"
+
+S="${WORKDIR}/QR-Code-generator-${PV}"
+
+LICENSE="MIT"
+
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+src_prepare() {
+	# Move the CMake files into the project root.
+	mv "${S}"/../qrcodegen-cmake-"${PV}"-cmake2/* "${S}"
```

${S} is the current directory in src_*, and PV does not need quoting, untested but could simplify as:
```suggestion
	mv ../qrcodegen-cmake-${PV}-cmake2/* . || die
```
(was also missing || die)

---

Yeah, I'll test that.

Had 'pkgcheck' complaining about at least one of them unquoted being, but didn't try which one...